src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -125,7 +125,8 @@ def _update_command_table_from_modules(args):
                 installed_command_modules = [modname for _, modname, _ in
                                              pkgutil.iter_modules(mods_ns_pkg.__path__)
                                              if modname not in BLACKLISTED_MODS]
-            except ImportError:
+            except ImportError as e:
+                logger.warning(e)
```

Comment:
@marstr the help test needed to load command modules and so the import on [line 124](https://github.com/Azure/azure-cli/pull/8325/files#diff-103131eb32c8ff55661bc902248c80a6R124) was failing because "azure.cli.command_modules" was not a tox dependency in our [tox.ini](https://github.com/Azure/azure-cli/blob/7a4ca585ed76dfaf4222659bad3e6acea57b2de9/src/azure-cli-core/tox.ini) file.